client-gateway/bot/passenger.py

```
@@ -283,74 +203,15 @@ async def handle_order_status(update: Update, context: ContextTypes.DEFAULT_TYPE
-            if call.data == "order_confirm":
-                chat_id = call.message.chat.id
-                order = user_orders.get(chat_id) or {}
-                # Log request before submitting to TripService
->>>>>>> main
                 logger.info(
                     "Trip request confirmed: chat_id=%s pickup=%s dropoff=%s comment=%s",
                     chat_id, order.get('pickup'), order.get('dropoff'), order.get('comment')
                 )
 
-<<<<<<< HEAD
+                start_time = time.time()
```

Flake8 (7.3.0)

[error] local variable 'start_time' is assigned to but never used

(F841)